the-codeless-code/fr-meowcate/case-230.txt

```diff
@@ -0,0 +1,84 @@
+Title: Tous ensemble désormais
+Tagline: Où l'on sous-traite certaines choses
+Lang: fr
+Translator: Meowcate
+
+Le [[Hungry Worm Clan|Clan du Ver Affamé]] construisait un site web
+pour un artisan qui fabriquait des outils agricoles spécialisés.
+La jeune maître [[Zjing]] était en pleine revue du code de ses trois
+développeurs.
+
+"Je ne comprends pas l'intérêt de la propriété /DateLimiteDeVente/ dans
+/AcheterProduit/ de votre panier," dit-elle aux trois.
+"Les pelles et les râteaux n'expirent pas."
+
+"C'est une propriété qui nous a été demandée par le
+[[Red Pebble Clan|Clan du Caillou Rouge]]," répondit le premier moine.
+"Ils construisent un système pour gérer les fermes d'un marchand de cerises,
+et ils ont l'intention d'utiliser notre service de commande plutôt que de
+concevoir le leur."
+
+"Quoi ?" s'exclama Zjink, incrédule. "Qui a suggéré /cela/ ?"
+
+"C'est vous," dit le deuxième moine, "N'avez-vous pas [[#227|dit à deux autres
+de nos clans]] que la réutilisation des services était supérieure au fait de
+copier et coller du code ?"
+
+"Certes, mais les besoins métier de /vos/ deux clans sont totalement
+différents !" se lamenta Zjing.
+"Par la suite, les acheteurs de cerises pourraient avoir besoin d'options
+pour la taille des boîtes, les livraisons réfrigérées, et l'assurance contre
+les insectes nuisibles.
+Tout cela pourrait nécessiter ses propres règles et calculs !"
+
+"Vous vous inquiétez pour un futur qui pourrait ne jamais voir le jour," opposa le
+troisième moine. "Et même si cela arrive, quel est le problème ? Plus nous
+implémentons de fonctionnalités, plus grandes sont les chances que nous
+puissions répondre aux besoins de nos autres clients."
+
+<i>"Autres</i> clients ?" demanda Zjing.
+
+"Deux autres Petits Clans ont montré de l'intérêt pour nos services,"
+dit le premier moine.
+
+"Pour des pièces détachées de monocycle et des costumes de mariage
+fantaisie," dit le deuxième moine.
+
+"Et j'ai commencé le design d'un méchanisme de plug-in pour gérer des
+cas imprévus," dit le troisième moine. "Au final le Temple économisera
+beaucoup de temps de développement."
+
+
+- - -
+
+Plus tard dans la semaine Zjing organisa une réunion des Petits Clans
```

Comment:
"meeting" has been changed to "réunion". Not "rencontre" as the room is made for this kind of event, nor "assemblée" because it sounds more like an annual general meeting.